x-pack/plugins/fleet/server/plugin.ts

```diff
@@ -313,6 +315,43 @@ export class FleetPlugin
             ui: ['read'],
           },
         },
+        subFeatures: [
+          {
+            name: 'Endpoint integration',
```

Comment:
One caveat with using subFeature is that you cannot enable read/write endpoint integration without enabling at least read integrations base privilege. This means the user will have read access to all packages. Is that a problem?

<img width="464" alt="image" src="https://user-images.githubusercontent.com/90178898/165950138-ad2c8cfa-3bb7-4015-a0ae-273f268d5ee2.png">